src/routes/Addons/Addon/Addon.js

```diff
@@ -84,9 +94,24 @@ const Addon = ({ className, id, name, version, logo, description, types, install
                 }
             </div>
             <div className={styles['buttons-container']}>
-                <Button className={installed ? styles['uninstall-button-container'] : styles['install-button-container']} title={installed ? t('ADDON_UNINSTALL') : t('ADDON_INSTALL')} tabIndex={-1} onClick={toggleButtonOnClick}>
-                    <div className={styles['label']}>{installed ? t('ADDON_UNINSTALL') : t('ADDON_INSTALL')}</div>
-                </Button>
+                <div className={styles['action-buttons-container']}>
+                    {
+                        !behaviorHints.configurationRequired && behaviorHints.configurable ?
```

Comment:
Site loads up until this point, error encountered with configurationRequired, it is not defined and throws a type error

![image](https://github.com/Stremio/stremio-web/assets/97270760/c9a35368-c3a9-4796-8409-2cd9acaa2c66)

Reply:
Error for reference:
![image](https://github.com/Stremio/stremio-web/assets/97270760/cbc9aeef-7f58-42f2-815b-7fec890f09ab)

Reply:
Did you do `npm install` after checkout ?